earn/src/components/markets/monitor/StatsTable.tsx

```diff
@@ -320,6 +323,11 @@ export default function StatsTable(props: { rows: StatsTableRowProps[] }) {
                   Reserve Factor
                 </Text>
               </th>
+              <th className='px-4 py-2 text-start whitespace-nowrap'>
+                <Text size='M' weight='bold'>
+                  Cardinality
+                </Text>
+              </th>
```

Comment:
Could we provide a description of what this is somewhere?

Reply:
Yes, I meant to ask you about adding some tooltips to a few of the columns.

Can you set up a component for that, and I can provide explanations?